components/ContactForm.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+
+const ContactForm: React.FC = () => {
+  const [resultMessage, setResultMessage] = useState<string | null>(null);
+
+  const handleFormSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault(); // Prevent the default form submission behavior
+
+    const form = e.currentTarget;
+    const formData = new FormData(form);
+
+    // Convert form data to an object
+    const object: { [key: string]: string } = {};
+    formData.forEach((value, key) => {
+      object[key] = value as string;
+    });
+
+    const json = JSON.stringify(object);
+
+    try {
+      const response = await fetch('https://api.web3forms.com/submit', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+          Accept: 'application/json',
+        },
+        body: json,
+      });
+
+      const jsonResponse = await response.json();
+      setResultMessage(jsonResponse.message);
+    } catch (error) {
+      console.error(error);
+      setResultMessage('Something went wrong!');
+    }
+
+    form.reset();
+  };
+
+  return (
+    <div className="container mx-auto">
+      <div className="m-7">
+        <form
+          className="flex-1 flex flex-col mx-auto w-full max-w-md justify-center text-left gap-2 text-foreground"
+          onSubmit={handleFormSubmit}
+        >
+          <input type="hidden" name="access_key" value="8347a957-fbc8-4284-9c75-27c5dbe46292" />
+          <input type="hidden" name="subject" value="New Submission from Web3Forms" />
+          <input type="checkbox" name="botcheck" style={{ display: 'none' }} />
+
+          <label className="text-md" htmlFor="email">
+            Email
+          </label>
+          <input
```

Comment:
Our input fields could probably all be required.

Reply:
Yeah good idea will add.